tests/run-make/llvm-location-discriminator-limit-dummy-span/rmake.rs

```diff
@@ -0,0 +1,58 @@
+//! Regression test for <https://github.com/rust-lang/rust/issues/135332>.
+//!
+//! We can't simply drop debuginfo location spans when LLVM's location discriminator value limit is
+//! reached. Otherwise, with `-Z verify-llvm-ir` and fat LTO, LLVM will report a broken module for
+//!
+//! ```text
+//! inlinable function call in a function with debug info must have a !dbg location
+//! ```
+
+//@ ignore-cross-compile
+//@ needs-dynamic-linking
+//@ only-nightly: requires unstable rustc flag
```

Comment:
Nit: I know I wrote this test, can you convert this into

```suggestion
//@ only-nightly (requires unstable rustc flag)
```

(I'd like to migrate off using colon form for comments on directives eventually, I accidentally wrote this.)